boards/ssrc/saluki/src/mpfs_spinor.c

```diff
@@ -0,0 +1,105 @@
+/****************************************************************************
+ * boards/risc-v/mpfs/icicle/src/mpfs_emmcsd.c
```

Comment:
Need to go through these file headers under board, for proper license text. this is coped from nuttx, and is probably not the right one for PX4

Reply:
ok